src/components/CheckoutForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import {
+  CardNumberElement,
+  CardExpiryElement,
+  CardCvcElement,
+  injectStripe
+} from "react-stripe-elements";
+import axios from "axios";
+import { Form, Button, Card, Container } from "semantic-ui-react";
+
+class CheckoutForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      success: "",
+      error: "",
+      renderStripeForm: true
+    };
+    this.submitPayment = this.submitPayment.bind(this);
+  }
+
+  async submitPayment() {
+    let token = await this.props.stripe.createToken();
```

Comment:
Is this the stripe token that gets sent to the backend?

Reply:
the createToken() is a function given to us by stripe. It 'tokenizes' the CC info given by the user, than that token is sent to the backend